client/src/components/accountInfo/accountInfo.jsx

```diff
@@ -1,20 +1,27 @@
-import React, { useState } from 'react';
-import { Button, TextField, Typography } from '@mui/material';
+import React, { useState, useEffect } from 'react';
+import { Button, TextField, Typography, Tooltip } from '@mui/material';
 import { Grid, Box } from '@mui/material';
 import { Link } from 'react-router-dom';
 import Avatar from '@mui/material/Avatar';
 import ProfilePic from '../../../dist/mockProfile.png';
+import axios from 'axios';
 
-function AccountInfo() {
+function AccountInfo(props) {
 
-   let mockDatabaseAccountNumber = 1234;
+   const [edit, setEdit] = useState(false);
 
-   let userInfo = {
-      firstName: "Daniel",
-      lastName: "Halper",
-      userName: "Dhalper",
-      email: "Dhalper@test.org",
-   }
+   let remainingFunds = 400;
+
+   const [userInfo, setUserInfo] = useState({
+      firstName: props.userInfo.firstName,
```

Comment:
Consider defensive programing and allowing undefined and null values to fail gracefully using optional chaining https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining